logic.py

```diff
@@ -1,6 +1,6 @@
 ## conditons when we have list of possible values 
```

Comment:
GPT summary of [ff48b2](https://github.com/SampatSuthar/ml-demo/blob/2ce9a211365f18cfb46299b15c0e532c3e8af178/logic.py#ff48b223391a0063edf03dca03d22da628bb8158) - [0f37af](https://github.com/SampatSuthar/ml-demo/blob/83037860d284bc693f5f6007e5bc84bcef2fd68c/logic.py#0f37af79ee671cc044fcbf2a54343096bc2e2834):
* Changed the value of the variable `state` from `Karnataka` to `Gujrat`
* Added a conditional statement to check if the value of `state` is in a list of possible values
* Added a line to print the petrol price of the given state